scripts/mail-alarm

```diff
@@ -34,66 +34,43 @@ FromLineOverride=YES
 ma_username="__dom0__mail_alarm"
 
 mail_language_pack_path = '/etc/xapi.d/mail-languages/'
+session = XenAPI.xapi_local()
```

Comment:
You now never login, I think you need to login at least once. At least for the purpose of making the session identifiable in xensource.log/audit.log as coming from the mail alarm script.